corehq/apps/cloudcare/templates/formplayer/case_list.html

```diff
@@ -63,7 +63,7 @@
       <% if (actions) { %>
         <% _.each(actions, function(action, index) { %>
         <div class="case-list-action-button btn-group formplayer-request">
-          <button type="button" class="btn btn-success btn-lg" data-index="<%- index %>"><%- action.text %></button>
+          <button type="button" class="btn <%- actionButtonStyle %> btn-lg " data-index="<%- index %>"><%- action.text %></button>
```

Comment:
My intent was that this just always use `btn-default`. As is, an app could have two case lists that have a case search / search again button, but it's green for one and white for the other.

These buttons arguably shouldn't have been green in the first place - it makes them stand out, but even for normal case lists, they're not the main action on this page (which is to select a case).

Reply:
gotcha-- switched it to always be defualt